File: AgentApplication/agent/src/main/java/com/example/agent/model/dto/JobOffer.java
package com.example.agent.model.dto;

import com.example.agent.model.JobPosition;
import lombok.AllArgsConstructor;
import lombok.Getter;
import lombok.NoArgsConstructor;
import lombok.Setter;

@Getter
@Setter
@NoArgsConstructor
@AllArgsConstructor
public class JobOffer {
    private String jobPosition;
    private String companyName;
    private String jobInfo;
    private String qualifications;
}

File: user-service/model/JobOffer.go
package model

type JobOffer struct {
	ID             uint   `json:"id"`
	JobPosition    string `json:"jobPosition"`
	CompanyName    string `json:"companyName"`
	JobInfo        string `json:"jobInfo"`
	Qualifications string `json:"qualifications"`
	ApiKey         string `json:"apiKey"`
}
